vendor/extensions/taggings/app/controllers/refinery/taggings/taggings_controller.rb

```diff
@@ -8,9 +8,10 @@ class TaggingsController < ::ApplicationController
       def index
         filters = [params[:content_type] || "everything", params[:topic_area] || "all topic areas"]
         topic_area_narrative = Refinery::Tags::Tag.all.find_by(title: filters[1]).narrative if filters[1] != "all topic areas"
-        filtered_taggings = Refinery::Taggings::Tagging.filter_taggings(filters)
-        filtered_taggings_sorted = filtered_taggings.sort { |a, b| b.sort_date <=> a.sort_date }.reverse
-        filtered_taggings_json = filtered_taggings_sorted.map {|t| TaggingSerializer.new(t).serializable_hash }
+        filtered_taggings_json = Refinery::Taggings::Tagging.filter_taggings(filters)
```

Comment:
Going through this commit-by-commit so this question may be answered elsewhere, but what are the differences we draw between "Tag"/"Tags" and "Tagging"/"Taggings"?

Reply:
Tagging/Taggings link the Tags to the other objects. It potentially could be refactored but this is tightly coupled throughout many files, so would be a heavy lift.